test/integration/test_integration.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-FileCopyrightText: 2017-2021 Contributors to the OpenSTF project <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+import unittest
+from test.unit.utils.data import TestData
+import os
+from openstef.pipeline.optimize_hyperparameters import optimize_hyperparameters_pipeline
+from openstef.pipeline.train_model import train_pipeline_common
+from openstef.pipeline.create_forecast import create_forecast_pipeline_core
+
+
+class TestIntegration(unittest.TestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.input_data = TestData.load("reference_sets/307-train-data.csv")
+        self.forecast_data = TestData.load("reference_sets/307-test-data.csv")
+        self.pj, self.modelspecs = TestData.get_prediction_job_and_modelspecs(pid=307)
+
+    def test_integration_happyflow(self):
+        """
+        Performs the integration test for the pipeline
+        1) optimize hyperparameters
+        2) train model
+        3) make prognoses
+        """
+
+        # 1) optimize hyperparameters
+        predefined_quantiles = (0.001, 0.5)
+        self.pj["quantiles"] = predefined_quantiles
+        parameters = optimize_hyperparameters_pipeline(
+            self.pj, self.input_data, "./trained_models", n_trials=2
+        )
+        self.modelspecs.hyper_params = parameters
+
+        # 2) train model, using the optimized hyperparameters
+        model, report, train_data, validation_data, test_data = train_pipeline_common(
+            self.pj, self.modelspecs, self.input_data, [0.25, 47.0]
+        )
+
+        # 3) create forecast, using the trained model
```

Comment:
Nice tests, could you split them up in separate methods? This way the pytest report will be much more readable